### DISCUSSION TITLE
pythonr writer optiions not respected

### DISCUSSION THREAD

**Discussion by Linch1** [openai/whisper#1683] [Q&A]
Hello, i'm trying to create the same result as the terminal command

```
whisper path.wav --word_timestamps True --max_line_width 22 --max_line_count 1 --highlight_words True
```

but i'm not getting the same result. I think that the code is almost complete but i'm not getting what option is missing or why it generates different results

```
import whisper
from whisper.utils import get_writer

def whisper_transcription(input_audio, whisper_model):

    model = whisper.load_model(whisper_model)
    result = model.transcribe(input_audio, verbose=True)
    output_directory = "."

    # Set some initial options values
    options = {
        'max_line_width': 22,
        'max_line_count': 1,
        'highlight_words': True
    }

   # Include options in call
    txt_writer = get_writer("txt", output_directory)
    txt_writer(result, input_audio, options)

    json_writer = get_writer("json",  output_directory)
    json_writer(result, input_audio, options)

    vtt_writer = get_writer("vtt", output_directory)
    vtt_writer(result, input_audio, options)

    tsv_writer = get_writer("tsv", output_directory)
    tsv_writer(result, input_audio, options)

    srt_writer = get_writer("srt", output_directory)
    srt_writer(result, input_audio, options)

audio_path = "path.wav"
whisper_transcription(audio_path, 'base')
```

**What am i missing?**

## Python output srt

1
00:00:00,000 --> 00:00:07,120
There's a really embarrassing moment. I definitely erased from my life. Once I arranged an underwater

## Command line output srt
1
00:00:00,000 --> 00:00:00,360
<u>Um,</u> there's a really

2
00:00:00,360 --> 00:00:00,540
Um, there's a really

3
00:00:00,540 --> 00:00:00,900
Um, <u>there's</u> a really

.. and so on, avoinding posting all the result since it useless.

**Comment by phineas-pta** [comment]
enable `word_timestamps`
